libdevcore/JsonUtils.cpp

```diff
@@ -31,3 +31,77 @@ void dev::validateFieldNames(json_spirit::mObject const& _obj, std::set<std::str
 			BOOST_THROW_EXCEPTION(UnknownField() << errinfo_comment(comment));
 		}
 }
+
+std::string dev::jsonTypeAsString(json_spirit::Value_type _type)
+{
+    switch (_type)
+    {
+    case json_spirit::obj_type:
+        return "json Object";
+    case json_spirit::array_type:
+        return "json Array";
+    case json_spirit::str_type:
+        return "json String";
+    case json_spirit::bool_type:
+        return "json Bool";
+    case json_spirit::int_type:
+        return "json Int";
+    case json_spirit::real_type:
+        return "json Real";
+    case json_spirit::null_type:
+        return "json Null";
+    default:
+        return "json n/a";
+    }
+}
+
+void dev::requireJsonFields(json_spirit::mObject const& _o, std::string const& _config,
```

Comment:
I'm not sure about having it in `libdevcore`

Reply:
the previous checks were put into this file. @pirapira ?

Reply:
If there is a better place, feel free to move.

Reply:
so we could remove jsonUtils from libdevcore. and keep it in libethereum. but I dont think its a better choise

Reply:
If `libdevcore` or `libethereum`, I'd vote for `libdevcore`.  This is not so particularly about Ethereum.